wire-ios/Wire-iOS/Sources/UserInterface/LaunchImage/LaunchImageViewController.swift

```diff
@@ -43,26 +43,21 @@ class LaunchImageViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        let loadedObjects = UINib(nibName: "ScreenCurtain", bundle: nil).instantiate(withOwner: nil, options: nil)
-
-        let nibView = loadedObjects.first as? UIView
-        nibView?.translatesAutoresizingMaskIntoConstraints = false
-        if let nibView = nibView {
-            view.addSubview(nibView)
-        }
-        if let nibView = nibView {
-            contentView = nibView
-        }
+        let contentView = UINib(nibName: "AppLock", bundle: nil)
+            .instantiate(withOwner: nil, options: nil)
+            .first as! UIView
```

Comment:
I would have kept the helper method on UIView since it's used twice

Reply:
Since a static method `UIView.shieldView()` is not what we want, I created a temporary helper: 4471295fd78e12893e8726d2557f040d45b6217a
On the long run I want to look into the `LaunchImageViewController` and if it's really needed.